vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -100,3 +115,8 @@ packages:
          - firewalld
          - python3-six
          - httpd-tools
+
+DAEMON_JSON_PATH: "{{ metal3_dir }}/vm-setup/roles/packages_installation/files"
```

Comment:
would it be sufficient not to point to the exact file name in this case?

Reply:
Could be. I think both are fine. Of course I can change it, but I don't see what would be the difference.

Reply:
Makes sense